engine/src/main/resources/application.conf

```diff
@@ -70,7 +70,8 @@ call-caching {
 
 backend {
   // Either "jes", "local", or "sge" (case insensitive)
-  backend = "local"
+  defaultBackend = "local"
+  backendsAllowed = [ "local" ]
```

Comment:
So this seems to conflict with @mcovarr's email about how to specify the backends... I'm guessing this is meant to change soon? (is there a ticket?)